### PR TITLE
Update Elixir Tutorial

### PR DESCRIPTION
**Pull request by burrelle**
Update to the Elixir tutorial

- Update Phoenix installer
- Remove brunch flag
- Update dependencies
- Update query to have description attribute and easier to follow
- Add mutation description field

Similar solutions to issues:
#940 
#941
#979

### CODE REVIEW

**Comment by monjohn** [comment]
I tried to follow the current version of the Elixir tutorial and got stuck with the same errors as in the issues listed above. I used the corrections in this PR and everything worked as expected.

This PR should be merged! 👍

File: content/backend/graphql-elixir/3-mutations.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Setting up mutations is as easy as queries, and you'll follow basically the same
 
 ```elixir(path=".../graphql-elixir/lib/community_web/schema.ex")
 mutation do
+  @desc “Create a new link”
```
Comment (monjohn) [inline content/backend/graphql-elixir/3-mutations.md]: These are smart quotes, but they need to be dumb quotes.

Comment (burrelle) [reply content/backend/graphql-elixir/3-mutations.md]: Good catch, thank you!

**Comment by soutot** [comment]
@idkjs I'm still getting the error. I could fix it by installing the following versions
```
{:absinthe_ecto, "~> 0.1.3"},
{:absinthe_plug, "~> 1.4.0"},
```

I don't know if @burrelle is still on it, it's been a month since your message. Could someone else edit this PR and merge it?

**Comment by burrelle** [comment]
@nikolasburk Would it be possible to get this PR merged?

**Comment by nikolasburk** [comment]
Hey @burrelle, sorry for the delay and thank you so much for taking the time to contribute. Just merged the PR 🎉

**Pull request merged**